### PR TITLE
ENCD-5133-calculate-for-analyses

### PR DESCRIPTION
**Pull request by yunhailuo**

### CODE REVIEW

File: src/encoded/schemas/experiment.json [hunk 1]
```diff
@@ -229,6 +229,58 @@
                             "type": "string",
                             "linkTo": "File"
                         }
+                    },
+                    "assemblies": {
+                        "title": "Assemblies",
+                        "description": "A list of genome assemblies on which this analsis is performed.",
```
Comment (hitz) [inline src/encoded/schemas/experiment.json]: s/analsis/analysis/

File: src/encoded/schemas/experiment.json [hunk 2]
```diff
@@ -229,6 +229,58 @@
...
+                        "comment": "[CALCULATED PROPERTY] Do not submit. This field is calculated from files in this analysis.",
+                        "type": "array",
+                        "notSubmittable": true,
+                        "items": {
+                            "type": "string"
+                        }
+                    },
+                    "genome_annotations": {
+                        "title": "Genome Annotations",
+                        "description": "A list of genome annotations on which this analsis is performed.",
```
Comment (hitz) [inline src/encoded/schemas/experiment.json]: s/analsis/analysis/

File: src/encoded/schemas/experiment.json [hunk 3]
```diff
@@ -229,6 +229,58 @@
...
+                        "comment": "[CALCULATED PROPERTY] Do not submit. This field is calculated from files in this analysis.",
+                        "type": "array",
+                        "notSubmittable": true,
+                        "items": {
+                            "type": "string"
+                        }
+                    },
+                    "pipelines": {
+                        "title": "Piplines",
```
Comment (hitz) [inline src/encoded/schemas/experiment.json]: s/Piplines/Pipelines/

File: src/encoded/schemas/experiment.json [hunk 4]
```diff
@@ -229,6 +229,58 @@
...
+                        "comment": "[CALCULATED PROPERTY] Do not submit. This field is calculated from files in this analysis.",
+                        "type": "array",
+                        "notSubmittable": true,
+                        "items": {
+                            "type": "string",
+                            "linkTo": "Pipeline"
+                        }
+                    },
+                    "pipeline_award_rfas": {
+                        "title": "Pipline awards",
```
Comment (hitz) [inline src/encoded/schemas/experiment.json]: s/Piplines/Pipelines/

File: src/encoded/schemas/experiment.json [hunk 5]
```diff
@@ -229,6 +229,58 @@
...
+                        "description": "A list of award bioproject phases to which pipelines that used to generate this analysis belong to.",
```
Comment (hitz) [inline src/encoded/schemas/experiment.json:266]: what is an award bioproject phase?

File: src/encoded/schemas/experiment.json [hunk 6]
```diff
@@ -229,6 +229,58 @@
...
+                        "comment": "[CALCULATED PROPERTY] Do not submit. This field is calculated from files in this analysis.",
+                        "type": "array",
+                        "notSubmittable": true,
+                        "items": {
+                            "type": "string"
+                        }
+                    },
+                    "pipeline_labs": {
```
Comment (hitz) [inline src/encoded/schemas/experiment.json:274]: Do we need an array of Labs?

**Review by hitz — CHANGES_REQUESTED** [review]
some typos

File: src/encoded/schemas/experiment.json (hunk 5 above)
Comment (yunhailuo) [reply src/encoded/schemas/experiment.json:266]: https://github.com/ENCODE-DCC/encoded/blob/dev/src/encoded/schemas/award.json#L179

File: src/encoded/schemas/experiment.json (hunk 6 above)
Comment (yunhailuo) [reply src/encoded/schemas/experiment.json:274]: @gabdank ?

Comment (gabdank) [reply src/encoded/schemas/experiment.json:274]: I think it should be a list of Labs. I think about all the lab-processed data as "non-uniform" while it is highly unlikely to have multiple labs processed data in one experiment, I think we should be able to handle this corner case?

File: src/encoded/schemas/experiment.json (hunk 5 above)
Comment (hitz) [reply src/encoded/schemas/experiment.json:266]: LOL

**Review by hitz — DISMISSED** [review]

**Comment by yunhailuo** [comment]
Rebased to solve merge one conflict in experiment changelog, @hitz .

**Review by hitz — APPROVED** [review]

**Pull request merged**